### PR TITLE
docs: add pt-BR translation to Helpers and Examples sections

### PR DESCRIPTION
**Pull request by mrspaiva**
🇧🇷

Added pt-BR translation for below items:

### Helpers
- Start Script
- Kill
- Wait for ports
- Wait for expected result
- Get pids
- Todo
- List files
- Log

### Examples
- Promises
- Parameterized tests
- Local server

### CODE REVIEW

File: website/i18n/pt-BR/docusaurus-plugin-content-docs/current/philosophy.mdx [hunk 1]
```diff
@@ -8,9 +8,7 @@ A filosofia do **Poku** se concentra na simplicidade e na eficiência, removendo
 
 <hr />
 
-<Heading as='h2' id='javascript-essence-for-tests-'>
-  Essência do JavaScript para testes 💡
-</Heading>
```
Comment (wellwelwel) [inline website/i18n/pt-BR/docusaurus-plugin-content-docs/current/philosophy.mdx:13]: It's intentional 🙋🏻‍♂️

**Review by wellwelwel — CHANGES_REQUESTED** [review]

File: website/i18n/pt-BR/docusaurus-plugin-content-docs/current/philosophy.mdx (hunk 1 above)
Comment (mrspaiva) [reply website/i18n/pt-BR/docusaurus-plugin-content-docs/current/philosophy.mdx:13]: Ok, I'll revert

File: website/i18n/pt-BR/docusaurus-plugin-content-docs/current/examples/local-server.mdx [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+
+<Tabs>
+  <TabItem default value='test/server.test.js'>
+
+```js
+import { assert, startScript } from 'poku';
+
+const server = await startScript('start', {
+  // Aguarde pelo "pronto" na saída do console
+  startAfter: 'pronto',
```
Comment (wellwelwel) [inline website/i18n/pt-BR/docusaurus-plugin-content-docs/current/examples/local-server.mdx:147]:
```suggestion
  startAfter: 'ready',
```

File: website/i18n/pt-BR/docusaurus-plugin-content-docs/current/examples/local-server.mdx [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+  <TabItem default value='server.js (Node.js)'>
+
+```js
+// highlight-start
+import { createServer } from 'node:http';
+
+createServer((_, res) => {
+  res.writeHead(200, { 'Content-Type': 'application/json' });
+  res.end(JSON.stringify({ name: 'Poku' }));
+}).listen(4000, () => console.log('pronto'));
```
Comment (wellwelwel) [inline website/i18n/pt-BR/docusaurus-plugin-content-docs/current/examples/local-server.mdx:24]:
```suggestion
}).listen(4000, () => console.log('ready'));
```

File: website/i18n/pt-BR/docusaurus-plugin-content-docs/current/examples/local-server.mdx [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+// highlight-start
+Bun.serve({
+  port: 4000,
+  fetch: () =>
+    new Response(JSON.stringify({ name: 'Poku' }), {
+      headers: { 'Content-Type': 'application/json' },
+    }),
+});
+
+console.log('pronto');
```
Comment (wellwelwel) [inline website/i18n/pt-BR/docusaurus-plugin-content-docs/current/examples/local-server.mdx:42]:
```suggestion
console.log('ready');
```

File: website/i18n/pt-BR/docusaurus-plugin-content-docs/current/examples/local-server.mdx [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+
+```ts
+// highlight-start
+Deno.serve({
+  port: 4000,
+  handler: () =>
+    new Response(JSON.stringify({ name: 'Poku' }), {
+      headers: { 'Content-Type': 'application/json' },
+    }),
+  onListen: () => console.log('pronto'),
```
Comment (wellwelwel) [inline website/i18n/pt-BR/docusaurus-plugin-content-docs/current/examples/local-server.mdx:57]:
```suggestion
  onListen: () => console.log('ready'),
```

**Review by wellwelwel — CHANGES_REQUESTED** [review]

**Review by wellwelwel — APPROVED** [review]

**Pull request merged**